Src/Common/Domain/service/null/NullOrganizationUserService.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace Ttu.Domain
+{
+    public class NullOrganizationUserService : IOrganizationUserService
+    {
+
+        public static IOrganizationUserService Singleton = new NullOrganizationUserService();
+
+        #region Constructors
+
+        protected NullOrganizationUserService()
+        {
+        }
+
+        #endregion
+
+        #region Public Methods
+
```

Comment:
usually have a line break between methods but i'm fine if you wanna just leave it.